templates/matching.jsx

```diff
@@ -38,7 +38,7 @@ export default function Matching(props) {
               'item',
               `item-${index}`,
               'js-matching-item',
-              displayItemAsCorrect ? 'is-correct' : 'is-incorrect'
+              !_canShowMarking ? '' : displayItemAsCorrect ? 'is-correct' : 'is-incorrect'
```

Comment:
Can you see if [_shouldShowMarking](https://github.com/adaptlearning/adapt-contrib-core/blob/master/js/models/questionModel.js#L388) works? It would seem to be a better fit here.

Reply:
of course, will test that out now and get back to you

Reply:
I tried using _shouldShowMarking but is seems to be set to false, even when I have the `Display Marking` setting set to true on the assessment and on the marking component. Meaning the marking doesnt get pulled through for each matching item.

Reply:
![image](https://user-images.githubusercontent.com/7974663/160835720-308bb63f-b479-4856-bae9-e50f9a0d7b82.png)
This works fine for me.
![image](https://user-images.githubusercontent.com/7974663/160837320-af12a188-12a9-4486-bcb1-f9a52e242167.png)